Tools/scripts/uploader.py

```diff
@@ -78,7 +79,7 @@
                   '/dev/serial/by-id/usb-Holybro*',
                   '/dev/tty.usbmodem*']
 
-if "cygwin" in _platform:
+if "cygwin" in _platform or "Microsoft" in subprocess.check_output(["cat", "/proc/version"]):
```

Comment:
what does this do if /proc/version doesn't exist? (eg. on macos)

Reply:
try it yourself! I imagine it would return whatever stdout would show such as can't find file or someything.. either way, it shouldn't contain the string "Microsoft" on real Linux :)

Reply:
I added an if exists check